src/inventory/gilded_rose.spec.js

```diff
@@ -1,11 +1,104 @@
 import { Item, updateQuality } from './gilded_rose';
 
-describe('`updateQuality`', () => {
+// ----Standard Item Quality----//
+describe('updateQuality', () => {
   it('deprecates the sell in by one for a Haunted Shoe', () => {
     const standardItem = new Item('Haunted Shoe', 10, 10);
     updateQuality([standardItem]);
     expect(standardItem.sell_in).toBe(9);
   });
 
-  it.todo('This is a good place for a good test!');
+  // ----Aged Brie Quality increases with age----//
+  // increase sell_in to 3 and quality to 1 --passes
+
+  it('increase quality of Aged Brie with age', () => {
+    const standardItem = new Item('Aged Brie', 3, 0);
+    updateQuality([standardItem]);
+    expect(standardItem.quality).toBe(1);
+  });
+
+  // ----Quality of item is never >50----//
+  // increase quality to 49 --passes
+
+  it('increase quality of Aged Brie with age', () => {
+    const standardItem = new Item('Aged Brie', 15, 49);
+    updateQuality([standardItem]);
+    expect(standardItem.quality).toBe(50);
+  });
+
+  // increase quality to 50 (shows that quality of Aged Brie can't go over 50)
+
+  it('Aged Brie quality cannot exceed 50', () => {
+    const standardItem = new Item('Aged Brie', 15, 51);
+    updateQuality([standardItem]);
+    expect(standardItem.quality).toBe(51);
+  });
+
+  // --- "Sulfuras"- never has to be sold nor does it decrease in quality---//
+  // Does not decrease in quality when I run it (test passed)
+
+  it('sulfuras quality does not decrease with age', () => {
+    const standardItem = new Item('Sulfuras, Hand of Ragnaros', 0, 80);
+    updateQuality([standardItem]);
+    expect(standardItem.quality).toBe(80);
+  });
+
+  // ----backstage passes----//
+  // increases in quality as it's sell_in value (age)decreases
+  it('increase quality of backstage passes with decreased sell_in', () => {
+    const standardItem = new Item('Backstage passes to a TAFKAL80ETC concert', 14, 20);
+    updateQuality([standardItem]);
+    expect(standardItem.quality).toBe(21);
+  });
+
+  // quality increases by 2 when there are 10 days or less
+  it('increase quality of backstage passes with decreased sell_in', () => {
```

Comment:
You can disregard this comment. Having duplicated code in tests is less of a concern because there is such an emphasis on making sure that tests are clear, straightforward, and only fail if there are failures in the source code.